internal/datastore/spanner/spanner.go

```diff
@@ -203,7 +218,7 @@ func (sd spannerDatastore) Features(_ context.Context) (*datastore.Features, err
 }
 
 func (sd spannerDatastore) Close() error {
-	sd.client.Close()
+	sd.readClient.Close()
```

Comment:
shouldn't it also close the `writeClient`?

```suggestion
	sd.readClient.Close()
	sd.writeClient.Close()	
```